server/models/Email.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectType, Field } from 'type-graphql';
+
+@ObjectType()
+export class Email {
+  @Field(() => String)
+  to: string[];
+
+  @Field(() => String)
+  subject: string;
+
+  @Field(() => String)
+  html: string;
+}
```

Comment:
Since this is only used in the mailer controller, I'd move it in the `server/controllers/Messages`

Reply:
As you suggested I changed the field type in Email.ts to an array of strings. I also moved the Email.ts file to the Messages folder. I was still getting a graphql error when I ran the mutation in the playground, but the messages were still being sent because I received all of them in my nodeMailer App. If I return only the subject from the mutation then it works fine.

![emailSent](https://user-images.githubusercontent.com/59596689/93953631-c24ee880-fd00-11ea-968c-c1f646ace406.png)

Reply:
The problem is that unlike REST, Graphql does runtime type checking, so if you say that `to` is going to be an array of strings and returns null/undefind it's going to error out. Check what is really returned from that Mailer service and change the types accordingly.

Reply:
I was able to successfully return the desired data by changing the names of the fields in Email.ts to match the properties returned by the MailerService. Not sure if that's the best way to do it, but it seemed simpler than changing anything in the MailerService.

![successReturn](https://user-images.githubusercontent.com/59596689/94211478-6c0bb200-fe86-11ea-89fb-d5fb4d87815b.png)